### ISSUE TITLE
Mutative methods on Non-mutative section

### ISSUE THREAD

**Issue by mfornet** [aurora-is-near/aurora-engine#376]
Methods on the [wasm engine interface](https://github.com/aurora-is-near/aurora-engine/blob/master/engine/src/lib.rs) are divided in those that [mutate the state](https://github.com/aurora-is-near/aurora-engine/blob/master/engine/src/lib.rs#L178), and those that [don't mutate the state](https://github.com/aurora-is-near/aurora-engine/blob/master/engine/src/lib.rs#L408).

There are methods (like [`ft_transfer`](https://github.com/aurora-is-near/aurora-engine/blob/master/engine/src/lib.rs#L559)) that mutates the state, but is in the non mutative section.

**Comment by mrLSD** [comment]
@mfornet when you mentioned **MUTATIVE** & **NONMUTATIVE** state section, you mean just grouping of methods after comments for an example:
```
    ///
    /// MUTATIVE METHODS
    ///
```

If it's true, `ft_trasfer` is in the **BENCHMARKING METHODS** section.
Am I clear that issue about regrouping methods to correct comment sections?

**Comment by mfornet** [comment]
> Am I clear that issue about regrouping methods to correct comment sections?

Yes, exactly this. This is only about putting the method in the section they belong. Maybe we should consider adding a section for NEP141 Standard implementation methods.

**Comment by joshuajbouw** [comment]
NEP-141 section is fine.
